client/src/shared/utils/interface.ts

```diff
@@ -136,6 +136,11 @@ export interface TabButtonProps {
   onClick?: () => void;
 }
 
+export interface TabProps {
+  title: string;
+  children?: ReactElement;
+}
+
```

Comment:
May I know why we need this in shared interfaces?, and I do not see u used it in your changes as well 🙇‍♂️